tests/common/direct-link-query-test.js

```diff
@@ -11,14 +11,21 @@ describe('common.direct-link-query', () => {
   it('returns the ID if the URL contains a #annotations:<ID> fragment', () => {
     var url = 'https://example.com/#annotations:1234';
     assert.deepEqual(directLinkQuery(url), {
-      id: '1234',
+      annotations: '1234',
     });
   });
 
-  it('returns the query if the URL contains a #annotations:<query> fragment', () => {
+  it('returns the query if the URL contains a #annotations:query:<query> fragment', () => {
     var url = 'https://example.com/#annotations:query:user%3Ajsmith';
     assert.deepEqual(directLinkQuery(url), {
       query: 'user:jsmith',
     });
   });
+
+  it('returns the group if the URL contains a #annotations:group:<id> fragment', () => {
```

Comment:
For consistency can we upper case the ID in `#anotations:group:<id>`? Perhaps "returns the **direct-linked group id** if the URL..."